### DISCUSSION TITLE
Namuwiki 관련해서 질문

### DISCUSSION THREAD

**Discussion by LDY0225** [List-KR/community#61] [AdGuard]
Namuwiki 관련해서 질문 드립니다. Adguard를 킨 상태에서 보고 있는데

1) '최근변경(Recentchanges)'이 안보이거나(화면에 나타나지 않습니다), 
2)문서,파일 등에 역링크된 것들이 보이지 않거나, 보이다가도 금방 사라진다거나
3)문서 편집 및 역링크가 바로바로 넘어가지 않거나 현재 보고 있는 문서로 보이지 않는다던지

등의 문제점이 보이는 것 같습니다.

혹시 대책이 없을까요? 또 이 현상은 Umanle가 Adguard 사용을 막기 위해 방해하는 거라 봐야 할까요?

**Comment by piquark6046** [comment]
해결된 것으로 보아 닫습니다.
아니라면, https://github.com/List-KR/NamuLink 에 issue를 열어주세요.
